modules/ROOT/pages/_partials/_define_component_attributes.adoc

```diff
@@ -8,7 +8,11 @@ ifdef::is_diag[_define_component_attributes.adoc]
 
 // Basic version
 :version: {major}.{minor}
+
+// Basic version for Vector Search
+:vs-version: {vs-major}.{vs-minor}
 // Base three digit version including optional tag
+:releasetag: beta.1
```

Comment:
I'd move this declaration to the antora.yml

e.g. The antora.yml is for things that we *expect to change* at a release, like configuration.

The partial is more like plumbing, to make sure that the things declared in the configuration get formatted in the right way.